handlers/workspaces.go

```diff
@@ -914,6 +914,30 @@ func (oh *workspaceHandler) GetFeaturesByWorkspaceUuid(w http.ResponseWriter, r
 	uuid := chi.URLParam(r, "workspace_uuid")
 	workspaceFeatures := oh.db.GetFeaturesByWorkspaceUuid(uuid, r)
 
+	for i, feature := range workspaceFeatures {
+
+		phases := oh.db.GetPhasesByFeatureUuid(feature.Uuid)
+
+		var totalCompleted, totalAssigned, totalOpen int
```

Comment:
@MahtabBukhari Make a count query from the DB
 e.g

```sh
func (db database) GetUserBountiesCount(personKey string, tabType string) int64 {
	var count int64

	query := db.db.Model(&Bounty{})
	if tabType == "bounties" {
		query.Where("owner_id", personKey)
	} else if tabType == "assigned" {
		query.Where("assignee", personKey)
	}

	query.Count(&count)
	return count
}
```
 instead of writing this loop.